plugin-template

```diff
@@ -52,6 +52,7 @@ DEFAULT_SETTINGS = {
     "pulp_env_azure": {},
     "pulp_env_gcp": {},
     "pulp_env_s3": {},
+    "pulp_openapi_generator_ref": None,
```

Comment:
How do we want to go about updating this?
Shall we try to keep it floating on latest main, or find the latest available tag, and only pin it down once we branch of a release branch?
Providing a default tag here will not really help updating it in the plugins, right?

Reply:
I like the floating on main and then pin once we branch a release branch. Sounds reasonable and straight-forward.

Reply:
Now we have a problem: After using the default branch on "main", this would pin the release branch to a probably older commit. That does not seem quite right, no?